Task_List_Kata/java/src/main/java/com/codurance/training/tasks/service/ShowService.java

```diff
@@ -0,0 +1,17 @@
+package com.codurance.training.tasks.service;
+
+import com.codurance.training.tasks.Task;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ShowService {
+
+    public void showByProject(Map<String, List<Task>> tasks);
+
+    public void showDueTodayTasks(Map<String, List<Task>> tasks);
+
+    public void showByDate(Map<String, List<Task>> tasks);
+
+    public void showByDeadline(Map<String, List<Task>> tasks);
```

Comment:
this violates interface seggregation principle.... please make different interfaces for each